EdFi.Ods.Utilities.Migration/Scripts/PgSql/02Upgrade/v50_to_v51/03 DeployJournal/v51UpdateDeployJournal.sql

```diff
@@ -9,34 +9,28 @@ ScriptName VARCHAR(255) NOT NULL
 
 INSERT INTO EdFiDeployJournal (ScriptName)
 VALUES
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.0010-Schemas.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.0020-Tables.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.0030-ForeignKeys.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.0040-IdColumnUniqueIndexes.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.0050-ExtendedProperties.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1002-AuthViews.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1003-CompositesHierarchicalViews.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1004-ViewIndexes.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1005-AddSQLTypes.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1020-Interoperability-Extension.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1021-OperationalContextView.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1022-SetVersion.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1030-AddSessionCascadeSupport.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1031-Create-SetSchoolYear-stored-procedure.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1034-RemoveDatesFromAuthViews.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1035-UpdateViewsToGoThroughStudendEdOrgResponsibiltyAssociation.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1070-RemoveDatesFromAuthViews.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1080-UpdateViewsToGoThroughStudendEdOrgResponsibiltyAssociation.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1090-MissingSecurityViews.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1100-AddMoreContextToEducationIdentifiersView.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1100-CorrectCommunityOrganizationAuthViews.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1120-RemoveCompositesHierarchicalViews.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1150-UpdatedAuthViewsToPreventDuplicatesFromStaffEdOrgAssignmentsAndEmployments.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Structure.Ods.1160-UpdateVersionTo510.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Data.Ods.0020-SchoolYears.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Data.Ods.1010-OperationalContexts.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Data.Ods.1020-Assign-Discriminator-Values.sql'),
-('EdFi.Ods.Standard.Artifacts.MsSql.Data.Ods.1040-Set-Default-SchoolYear.sql');
+('EdFi.Ods.Standard.Artifacts.PgSql.Structure.Ods.0010-Schemas.sql'),
+('EdFi.Ods.Standard.Artifacts.PgSql.Structure.Ods.0011-Extensions.sql'),
+('EdFi.Ods.Standard.Artifacts.PgSql.Structure.Ods.0020-Tables.sql'),
+('EdFi.Ods.Standard.Artifacts.PgSql.Structure.Ods.0030-ForeignKeys.sql'),
```

Comment:
should this be on a separate ticket?

Reply:
Ticket merged now , I did rebase main also .